docs/conf.py

```diff
@@ -166,22 +156,16 @@
 man_pages = [('index', project.lower(), project + u' Documentation', [author], 1)]
 
 # -- Swap to Napoleon ---------------------------------------------------------
+# Remove numpydoc
+extensions.remove('numpydoc')
 extensions.append('sphinx.ext.napoleon')
 
 # Disable having a separate return type row
 napoleon_use_rtype = False
 # Disable google style docstrings
 napoleon_google_docstring = False
 
-# -- Options for the edit_on_github extension ----------------------------------------
```

Comment:
This comment should still be here.

Reply:
Done

Reply:
This was moved to the wrong part of the file?